tasks/main.yml

```diff
@@ -38,3 +40,16 @@
     state: present
   environment:
     - ACCEPT_EULA: Y
+
+- name: Gather the package facts
+  ansible.builtin.package_facts:
+    manager: auto
+
+- name: Ensure that firewalld is configured
+  firewalld:
+    zone: public
+    port: "{{ mssql_db_port }}/tcp"
+    permanent: true
+    state: enabled
+  notify: restart firewalld
+  when: '"firewalld" in ansible_facts.packages'
```

Comment:
I don't think we should have firewall in the role, at least, not yet.
Several other roles have firewall (and related selinux) requirements, and for those the README just tells the user what to do: https://github.com/linux-system-roles/logging/#port-and-selinux
https://github.com/linux-system-roles/metrics/ "This role does not configure for remote access to these services such as through a firewall. See https://github.com/linux-system-roles/firewall"
https://github.com/linux-system-roles/vpn#firewall

So I think the mssql role should do something similar in the README

However - in the CI tests, we may want to add firewall and selinux setup - otoh, we don't currently have multi-host testing in CI

Reply:
I've added a requirement to README. Do CI machines have the firewall enabled. MSSQL does not use selinux, just that a port must be opened so that others can access the MSSQL server.